spec/spec.md

```diff
@@ -760,10 +799,9 @@ const ERROR_INVALID_PAYLOAD = -3
 const ERROR_NOT_SIGNED = -4
 const ERROR_NOT_COMMITTED = -5
 const ERROR_TOO_MANY_PAYLOADS = -6
+const ERROR_CLUSTER_NOT_SUPPORTED = -7
 
 const ERROR_ATTEST_ORIGIN_ANDROID = -100
-
-const MAX_TRANSACTION_SZ = 1232
```

Comment:
The MWA protocol doesn't need to define this explicitly. A wallet will refuse to sign an oversized transaction payload. Not having this in the MWA spec means one less magic constant to keep in sync.